### ISSUE TITLE
Code for validation loss

### ISSUE THREAD

**Issue by rafaelvalle** [da03/Attention-OCR#33]
Looking at the code I only see training loss. Is there any routine that saves both training and validation loss?
